CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
 - Exposes `ScreenSpaceCameraController.zoomFactor` to allow adjusting the zoom factor (speed). [#9145](https://github.com/CesiumGS/cesium/pull/9145)
 
+- Removed `jitter` option from `VoxelPrimitive.js` and related test code in `VoxelPrimitiveSpec.js`. [#11913](https://github.com/CesiumGS/cesium/issues/11913)
```

Comment:
Can you please move this into the breaking changes section?

Reply:
Hello @ankittrehan2000
I adjusted the CHANGES.md and also removed the following additional jitter code from VoxelRenderResources.js

if (primitive._jitter) {
    shaderBuilder.addDefine("JITTER", undefined, ShaderDestination.FRAGMENT);
  }
  
  Thank you for your feedback!